OrbitGl/ProcessListManager.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORIBIT_GL_PROCESS_MANAGER_H_
+#define ORIBIT_GL_PROCESS_MANAGER_H_
+
+#include <memory>
+#include <string>
+#include <thread>
+
+#include "absl/synchronization/mutex.h"
+#include "absl/time/time.h"
+#include "grpcpp/grpcpp.h"
+#include "process.pb.h"
+
+// This class is resposible for maintaining
+// process list. It periodically updates it
+// and calls callback to notify listeners when
+// the list is updated.
+//
+// Usage example:
+//
+// auto manager = ProcessListManager::Create(...);
+// auto callback = [&](const std::vector<ProcessInfo>& process_list) {
+//   // Update process list in UI
+// }
+// manager.SetCallback(callback);
+//
+// manager.Start();
+//
+// To orderly shutdown the manager use following:
+//
+// manager.Shutdown();
```

Comment:
Why are Shutdown and Wait separate steps, could Shutdown call Wait so that the user doesn't have to?

Reply:
good point..

Reply:
done